std/process.d

```diff
@@ -1280,7 +1291,12 @@ private Pid spawnProcessImpl(scope const(char)[] commandLine,
     if (!CreateProcessW(null, commandLine.tempCStringW().buffPtr,
                         null, null, true, dwCreationFlags,
                         envz, workDir.length ? pworkDir : null, &startinfo, &pi))
-        throw ProcessException.newFromLastError("Failed to spawn new process");
+    {
+        if (GetLastError() == ERROR_FILE_NOT_FOUND)
+            throw new ProcessException(text("Executable file not found: ", program));
```

Comment:
What happens if a DLL or other resource is missing?

Reply:
For a missing DLL, it doesn't seem to result in CreateProcessW failing (and instead make the child process exit with != 0). Tested with a `pythonw.exe` and taking away its `python37.dll`.